--- A/maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/changelog/SvnChangeLogConsumerTest.java	2010-09-08 14:28:29.459743001 +0200
+++ B/maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/changelog/SvnChangeLogConsumerTest.java	2010-09-08 14:30:03.766743001 +0200
@@ -34,6 +34,7 @@
 import org.apache.maven.scm.ChangeSet;
 import org.apache.maven.scm.log.DefaultLog;
 import org.codehaus.plexus.PlexusTestCase;
+import org.codehaus.plexus.DefaultPlexusContainer;
 import org.codehaus.plexus.logging.Logger;
 
 /**
@@ -52,7 +53,7 @@
     {
         super.setUp();
 
-        logger = getContainer().getLogger();
+        logger = ((DefaultPlexusContainer) getContainer()).getLogger();
         consumer = new SvnChangeLogConsumer( new DefaultLog(), null );
     }
     
