--- A/maven-scm-providers/maven-scm-provider-vss/src/test/java/org/apache/maven/scm/provider/vss/commands/status/VssStatusConsumerTest.java	2010-09-08 15:09:28.447743001 +0200
+++ B/maven-scm-providers/maven-scm-provider-vss/src/test/java/org/apache/maven/scm/provider/vss/commands/status/VssStatusConsumerTest.java	2010-09-08 15:11:02.098743001 +0200
@@ -31,6 +31,7 @@
 import org.apache.maven.scm.provider.vss.repository.VssScmProviderRepository;
 import org.apache.maven.scm.repository.ScmRepository;
 import org.apache.maven.scm.repository.ScmRepositoryException;
+import org.codehaus.plexus.DefaultPlexusContainer;
 import org.codehaus.plexus.util.IOUtil;
 
 /**
@@ -50,7 +51,7 @@
     {
         super.setUp();
         scmManager = getScmManager();
-        logger = getContainer().getLogger();
+        logger = ((DefaultPlexusContainer) getContainer()).getLogger();
     }
 
     public void testConsumeLine()
